demo/webpack.config.js

```diff
@@ -35,12 +60,12 @@ module.exports = {
 							modules: false,
 							loose: true
 						}],
-						require.resolve('@babel/preset-react'),
+						[require.resolve('@babel/preset-react')],
```

Comment:
I added these here just cuz I found it easier to read when al the plugins and presets lined up in a nice column. yay alignment!